src/client/components/Product/Product.styles.css

```diff
@@ -22,15 +27,17 @@
 }
 
 .product-add-button {
-  background: #36723a;
+  background: #687808;
```

Comment:
Looks way better with this changes 👍 . If you can add also something like the following so we can see when the button is clicked: 
`.product-add-button:active {
  background-color: #414b0b;
}
`